moc-health-check/config.cfg

```diff
@@ -0,0 +1,22 @@
+[ConfigInfo]
+
+# Size of the file that is sent to Pfioh in MB (1,2,3...)
+SIZE=6
+
+# Number of attempts required to check success rate
+RANGE=2

```

Comment:
The name range doesn't really seem to indicate what it's doing.  How about MAX_ATTEMPTS?